cryptography/src/bls12381/dkg/ops.rs

```diff
@@ -95,16 +95,16 @@ pub fn verify_share<V: Variant>(
 
 /// Construct a public polynomial by summing a vector of commitments.
 pub fn construct_public<V: Variant>(
-    commitments: Vec<poly::Public<V>>,
+    commitments: &BTreeMap<u32, poly::Public<V>>,
```

Comment:
We can use itiap (`commitments: impl Iterator<Item = poly::Public<V>>`) to avoid mandating the caller to construct a map when it's not necessary

Reply:
I thought about doing this but got lazy. Will add now.